### ISSUE TITLE
[Bug]: Error `GrantEntity has no db_name field` will occur when using some Role apis

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#25113]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20230625-4ac391db 
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):2.4.0.dev79
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Error `GrantEntity has no db_name field` will occur when using some Role apis.

```
[2023-06-25 16:05:38 - ERROR - pymilvus.decorators]: Unexpected error: [grant_privilege], Protocol message GrantEntity has no "db_name" field., <Time: {'RPC start': '2023-06-25 16:05:38.009994', 'Exception': '2023-06-25 16:05:38.010110'}> (decorators.py:120)
[2023-06-25 16:05:38 - ERROR - ci_test]: Traceback (most recent call last):
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/decorators.py", line 105, in handler
    return func(*args, **kwargs)
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/decorators.py", line 136, in handler
    ret = func(self, *args, **kwargs)
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/decorators.py", line 87, in handler
    raise e
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/client/grpc_handler.py", line 1285, in grant_privilege
    req = Prepare.operate_privilege_request(role_name, object, object_name, privilege, db_name,
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/client/prepare.py", line 806, in operate_privilege_request
    entity=milvus_types.GrantEntity(role=milvus_types.RoleEntity(name=role_name),
ValueError: Protocol message GrantEntity has no "db_name" field.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/zilliz/nico/milvus/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
    res = func(*args, **_kwargs)
  File "/Users/zilliz/nico/milvus/tests/python_client/utils/api_request.py", line 57, in api_request
    return func(*arg, **kwargs)
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/orm/role.py", line 152, in grant
    return self._get_connection().grant_privilege(self._name, object, object_name, privilege, db_name)
  File "/Users/zilliz/virtual-environment/milvus2/lib/python3.10/site-packages/pymilvus/decorators.py", line 121, in handler
    raise MilvusException(message=f"Unexpected error, message=<{str(e)}>") from e
pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=Unexpected error, message=<Protocol message GrantEntity has no "db_name" field.>)>
 (api_request.py:39)
[2023-06-25 16:05:38 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=Unexpected error, message=<Protocol message GrantEntity has no "db_name" field.>)> (api_request.py:40)

```

It seems related to db_name in these apis:
https://github.com/milvus-io/pymilvus/blob/cadf7b4a09c9a0a2d808c6daef6f493502578e9a/pymilvus/orm/role.py#L134
https://github.com/milvus-io/pymilvus/blob/cadf7b4a09c9a0a2d808c6daef6f493502578e9a/pymilvus/orm/role.py#L154
https://github.com/milvus-io/pymilvus/blob/cadf7b4a09c9a0a2d808c6daef6f493502578e9a/pymilvus/orm/role.py#L174
https://github.com/milvus-io/pymilvus/blob/cadf7b4a09c9a0a2d808c6daef6f493502578e9a/pymilvus/orm/role.py#L197

### Expected Behavior

pass

### Steps To Reproduce

```markdown
One case for example:
test_clear_roles
```

**Comment by XuanYang-cn** [comment]
should be fixed at pymilvus-2.4.0.dev81
/unassign
/assign @NicoYuan1986

**Comment by NicoYuan1986** [comment]
Fixed.
